packages/sources/layer2-sequencer-health/schemas/env.json

```diff
@@ -4,9 +4,29 @@
   "required": [],
   "type": "object",
   "properties": {
-    "DELTA": {
+    "ARBITRUM_DELTA": {
       "type": "number",
-      "default": 180000
```

Comment:
I would keep this original delta, I'm thinking that maybe some NOP could have it set to something, we change the adapter, they update without setting the new variables and we get into a state they did not expect. So the path would be chain_specific env var -> delta env var -> hardcoded default

Reply:
@mohamed-mehany explained in DMs, makes sense to separate defaults, NOPs will be aware of this change